signalsets/v3/default.json

```diff
@@ -1,4 +1,35 @@
 { "commands": [
-
+{ "hdr": "732", "dbg": true, "cmd": {"22": "D928"}, "freq": 1,
+  "signals": [
+    {"id": "FPACE_PARKED", "path": "Movement", "fmt": { "len": 8, "max": 1, "unit": "noyes" }, "name": "Parked?"}
```

Comment:
Are we sure this is using all 8 bits? If it's just using the last bit, consider adding a bix: 7